src/doc/en/developer/git_trac.rst

```diff
@@ -76,8 +76,8 @@ to the Sage directory and tell ``git trac`` about your trac account::
 
     [user@localhost sage]$ git trac config --user USERNAME --pass 'PASSWORD'
     Trac xmlrpc URL:
-        https://trac.sagemath.org/xmlrpc (anonymous)
```

Comment:
The changes in this file are best reverted.

Reply:
Done.